opsdroid/connector/gitter/__init__.py

```diff
@@ -0,0 +1,80 @@
+
+import logging
+import aiohttp
+import asyncio
+import json
+import urllib
+
+from opsdroid.connector import Connector,register_event
+from opsdroid.events import Message
+
+_LOGGER = logging.getLogger(__name__)
+GITTER_STREAM_API = "https://stream.gitter.im/v1/rooms"
+
+
+class ConnectorGitter(Connector):
+
+  def __init__(self, config, opsdroid=None):
+    """Create the connector."""
+    super().__init__(config, opsdroid=opsdroid)
+    _LOGGER.debug("Starting Gitter connector")
+    self.name = "gitter"
+    self.session = None
+    self.response = None
+    self.bot_name = self.config.get("bot-name", 'opsdroid')
+    self.room_id = self.config.get("room-id")
+    self.access_token = self.config.get("access-token")
+    self.update_interval = 1
+    self.opsdroid = opsdroid
+
+  async def connect(self):
+
+    # Create connection object with chat library
+    _LOGGER.debug("Connecting with gitter stream")
+    self.session = aiohttp.ClientSession()
+    gitter_url = self.build_url(GITTER_STREAM_API,self.room_id,"chatMessages", access_token = self.access_token)
+    _LOGGER.debug("Gitter stream url %s",gitter_url )
+    self.response = await self.session.get(gitter_url)
+
+  def build_url(self,base_url , *res, **params):
+    url = base_url
+    for r in res:
+        url = '{}/{}'.format(url, r)
+    if params:
+        url = '{}?{}'.format(url, urllib.parse.urlencode(params))
+    return url
+
+  async def listen(self):
+    _LOGGER.debug("Listening with gitter stream")
+    while True:
+      await asyncio.sleep(self.update_interval)
+      async for data in self.response.content.iter_chunked(1024):
+        _LOGGER.info(data)
+        try:
+            data = await self.parse_message(data)
+            if data !=None:
+              message = Message(
+              data["text"],
+              data["fromUser"]["id"],
+              self)
+              await self.opsdroid.parse(message)
+        except Exception as err:
+          _LOGGER.error("Unable to parse message %s", data)
```

Comment:
error i am facing is ::
`Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/opsdroid/core.py", line 346, in run_skill
    await skill(self, config, message)
  File "/root/.local/share/opsdroid/opsdroid-modules/skill/hello/__init__.py", line 9, in hello
    await message.respond(text)
  File "/usr/local/lib/python3.6/site-packages/opsdroid/events.py", line 187, in respond
    if 'thinking-delay' in self.connector.configuration or \
AttributeError: 'NoneType' object has no attribute 'configuration'`